attestation-service/verifier/src/se/ibmse.rs

```diff
@@ -0,0 +1,318 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use log::{debug, warn};
+use openssl::{
+    encrypt::{Decrypter, Encrypter},
+    pkey::{PKey, Private, Public},
+    rsa::{Padding, Rsa},
+};
+use pv::{
+    attest::{
+        AdditionalData, AttestationFlags, AttestationItems, AttestationMeasAlg,
+        AttestationMeasurement, AttestationRequest, AttestationVersion,
+    },
+    misc::{open_file, read_certs, read_file},
+    request::{BootHdrTags, CertVerifier, HkdVerifier, ReqEncrCtx, Request, SymKeyType},
+    uv::ConfigUid,
+};
+use serde::{Deserialize, Serialize};
+use serde_json;
+use serde_with::{base64::Base64, serde_as};
+use std::{fs::File, io::Read};
+
+fn encrypt_measurement_key(key: &[u8], rsa_public_key: &PKey<Public>) -> Vec<u8> {
+    let mut encrypter = Encrypter::new(rsa_public_key).unwrap();
+    encrypter.set_rsa_padding(Padding::PKCS1).unwrap();
+
+    let buffer_len = encrypter.encrypt_len(key).unwrap();
+    let mut encrypted_hmac_key = vec![0; buffer_len];
+    let len = encrypter.encrypt(key, &mut encrypted_hmac_key).unwrap();
+    encrypted_hmac_key.truncate(len);
+
+    encrypted_hmac_key
+}
+
+fn decrypt_measurement_key(key: &[u8], rsa_private_key: &PKey<Private>) -> Vec<u8> {
+    let mut decrypter = Decrypter::new(rsa_private_key).unwrap();
```

Comment:
Let's try to avoid all the `unwrap()`s and replace with `anyhow` error handling.